sparse_strips/vello_hybrid/shaders/sparse_strip_renderer.wgsl

```diff
@@ -129,3 +130,16 @@ fn unpack_alphas_from_channel(rgba: vec4<u32>, channel_index: u32) -> u32 {
         default: { return rgba.x; }
     }
 }
+
+// Polyfills `unpack4x8unorm`.
+//
+// Downlevel targets do not support native WGSL `unpack4x8unorm`.
+fn unpack4x8unorm(rgba_packed: u32) -> vec4<f32> {
+    // Extract each byte and convert to float in range [0,1]
+    return vec4<f32>(
+        f32((rgba_packed >> 0u) & 0xFFu) / 255.0,  // r
+        f32((rgba_packed >> 8u) & 0xFFu) / 255.0,  // g
+        f32((rgba_packed >> 16u) & 0xFFu) / 255.0, // b
+        f32((rgba_packed >> 24u) & 0xFFu) / 255.0  // a
+    );
+}
```

Comment:
[nit]: trailing newline missing

Reply:
(You should be able to configure your editor to add these automatically)

Reply:
My IDE didn't like `.wgsl` 😅